utils/json-schemas/performance_notification.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://frinx.io/performance_notification.json",
+  "title": "Performance monitoring notification",
+  "description": "Schema for performance monitoring notifications",
+  "type": "object",
+  "properties": {
+    "node_id": {
+      "description": "The name of the node.",
+      "type": "string"
+    },
+    "identifier": {
+      "description": "Identifier/path to data.",
+      "type": "string"
+    },
+    "event_time": {
+      "description": "The timestamp when notification arrived.",
+      "type": "string"
+    },
+    "body": {
+      "description": "The data.",
+      "type": "string"
```

Comment:
are these events sent as escaped jsons? if not, then 'object' type is better:
https://json-schema.org/understanding-json-schema/reference/object

Reply:
Probably unescaped. But now I have only notifications with numbers like this:
    "body": {
        "nokia-state:summary": {
            "available-memory": "2297430016",
            "current-total-size": "1862270976",
            "total-in-use": "1679717632"
        }
    }
I changed field to object